website/docs/wallet/introduction.md

```diff
@@ -63,7 +63,13 @@ HD wallets are password protected via a high-entropy, strong password, and allow
 
 [Create and use an HD wallet](https://docs.prylabs.network/docs/wallet/deterministic)
 
-## Remote signing wallet
+## Remote signing wallet (marked for removal) 
+
+:::caution
+
+This feature will be removed in a future hardfork and has been superseded by [web3signer](web3signer.md)
```

Comment:
Can we say explicitly that it will be the next hardfork, Capella, and version v4 of Prysm?